swan-lake/integration-tutorials/sending-a-message-to-a-service.md

```diff
@@ -121,7 +147,7 @@ Follow the instructions given in this section to develop the service.
         Doctor[]|http:ClientError resp = queryDoctorEP->/[category];
         ```
 
-    - Use the `is` check to decide the response based on the response to the client call. If the client call was successful and the respond payload was an array of `Doctor` records (as expected), then, directly return the array from the resource. If the request failed, send an `http:NotFound` response if the client call failed with a `4xx` status code or return an `http:InternalServerError` response for other failures.
+    - Use the `is` check to decide the response based on the response to the client call. If the client call was successful and the respond payload was an array of `Doctor` records (as expected), then, directly return the array from the resource. If the request fails, send an `http:NotFound` response if the client call failed with a `4xx` status code or return an `http:InternalServerError` response for other failures.
 
         ```ballerina
```

Comment:
Why having the code here again? Since it is already given at the beginning of step 7, this is duplicated content.

Reply:
IMO, Doesn't this make the code more understandable by the user?